cmd/fitgen/internal/profile/generator_test.go

```diff
@@ -195,6 +195,7 @@ var sdks = []sdk{
 	{20, 14},
 	{20, 27},
 	{20, 43},
+	{21, 40},
```

Comment:
Not sure if this change should be included, but I just thought it would be nice to add 21.xx to the test suite.
Have no idea how to properly validate the "golden" file, so I just included the one that test has generated.

Reply:
Well, good question... I kind of hard to validate the initial version, but I have mainly used it earlier to spot any changes/bugs going forward.